### ISSUE TITLE
dbind warnings and xsession errors

### ISSUE THREAD

**Issue by d451d15** [canonical/lightdm#392]
After a recent update, the mail client evolution stopped working, with the error

(evolution:2978): dbind-WARNING **: 10:04:20.105: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/134/at-spi/bus_0: Permission denied.

User 134 is lightdm. Also many other errors appear in ~/Xsession-errors for the following apps (however these apps still work):

(thunar:2297): dbind-WARNING **: 11:55:41.927: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/134/at-spi/bus_0: Permission denied

(mousepad:4189): dbind-WARNING **: 12:31:52.235: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/134/at-spi/bus_0: Permission denied

(xfce4-terminal:4243): dbind-WARNING **: 12:32:44.614: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/134/at-spi/bus_0: Permission denied.

Switching to a different display manager resolves the problem. Everything worked fine before the system update.
Is there a tweak in lightdm configuration to solve this? I am using Gentoo linux.

A discussion of the problem can be found at https://forums.gentoo.org/viewtopic-t-1172784-highlight-.html

**Comment by d451d15** [comment]
I closed the issue because I don't think it is a lightdm problem
